package.json

```diff
@@ -51,6 +53,9 @@
     "mocha": "2.3.x",
     "warehouse-models": "^5.1.1",
     "request": "2.65.x",
+    "sinon": "^1.15.4",
+    "sinon-spy-react": "^2.0.0",
+    "sinon-chai": "^2.8.0",
```

Comment:
`assume` is used as test module here, there should be a `sinon-assume` or `assume-sinon` don't recall exactly. However I think you can go without both `sinon-chai` and `sinon-spy-react` as they are not used elsewhere.